src/atoms/gv-switch.js

```diff
@@ -39,6 +39,7 @@ import { dispatchCustomEvent } from '../lib/events';
  * @cssprop {Color} [--gv-switch-on--bgc=var(--gv-theme-color, #5a7684)] - On background color
  * @cssprop {Color} [--gv-switch-off--bgc=var(--gv-theme-neutral-color-dark, #bfbfbf)] - Off background color
  * @cssprop {Color} [--gv-switch--bgc=var(--gv-theme-neutral-color-lightest, #ffffff)] - Switch background color
+ * @cssprop {String} [--gv-switch--text-align=left] - Switch label text alignment
```

Comment:
We use this https://docs.emmet.io/cheat-sheet/ for poperties name suffix. Here it gives `--gv-switch--ta`.
Also when we add or change var css, we need to run `generate:theme` task.
https://components.gravitee.io/?path=/story/documentation-how-to-s-components--page

Reply:
Thanks, it's fixed.
I take note.